system-status.sh

```diff
@@ -38,3 +38,13 @@ log minikube_ssh journalctl --boot
 # filesystem status for host and VM
 log df -hT
 log minikube_ssh df -hT
+
+# fetch all logs from /var/lib/rook in the VM and write them to stdout
+log minikube_ssh sudo tar c /var/lib/rook | tar xvO
+
+# gets status of the Rook deployment
+kubectl -n rook-ceph get pods
```

Comment:
Similar to the log() function that tracks which command execution is starting and ending, can we do something similar for kubectl commands so that it is easier to debug? Maybe just printing the command we run?

Reply:
oh, yes, missed that!